config/Schema/acl.sql

```diff
@@ -0,0 +1,52 @@
+# $Id$
+#
+# Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+#
+# Licensed under The MIT License
+# For full copyright and license information, please see the LICENSE.txt
+# Redistributions of files must retain the above copyright notice.
+# MIT License (http://www.opensource.org/licenses/mit-license.php)
+
+CREATE TABLE acos (
+	id INTEGER(10) UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
Since this plugin can be used with any database cakephp supports, I would suggest including a migrations file instead of a SQL. This will unfortunately only work with MySQL